eureka/S3_data_reduction/wfc3_1reduce.py

```diff
@@ -1,954 +0,0 @@
-#! /usr/bin/env python
```

Comment:
Did you intend to delete everything from this file ?

Reply:
Yeah, this file was first uploaded when Eureka was made and hasn't been used since then